sdk/python/kfp/components/_component_store.py

```diff
@@ -21,10 +22,13 @@ class ComponentStore:
 
     def __init__(self,
                  local_search_paths=None,
+                 uri_search_template=None,
```

Comment:
This needs to be added to the end, otherwise it's a breaking change for those who use positional arguments.

Besides, can you add this and the existing arguments in the docstring?

Reply:
Makes sense. Logically I wanted to keep the URI arguments together, but I take your point about compatibility.

Reply:
Done. The docstring is a bit repetitious now between the class and the constructor, let me know if you'd prefer it to look different: I tried to follow the Google style guidelines on docstrings for classes.